docs/RtspStreamingGuide.md

```diff
@@ -0,0 +1,371 @@
+# USB Camera Device Service RTSP Streaming Guide
+
+## Contents
```

Comment:
Make sure to add a double space after each line so there is a line break for each item